source/client/gui/components/TextInputBox.cpp

```diff
@@ -47,117 +45,111 @@ void TextInputBox::setEnabled(bool bEnabled)
 	m_bEnabled = true;
 }
 
-void TextInputBox::keyPressed(Minecraft* minecraft, int key)
+void TextInputBox::keyPressed(int key)
 {
 	if (!m_bFocused)
-		return;
-
-	bool bShiftPressed = minecraft->platform()->shiftPressed();
-	
-#ifndef HANDLE_CHARS_SEPARATELY
```

Comment:
`HANDLE_CHARS_SEPARATELY` is not always available.

Reply:
Done!